fetch_from_legacy.py

```diff
@@ -60,6 +60,12 @@ def handle_endtag(self, tag):
 context.check_hostname = False
 context.verify_mode = ssl.CERT_NONE
 
+# Extract out username/password from index_url, if present.
+cleaned_url = urlparse(index_url)
```

Comment:
I think cleaned_url is strange naming. It isn't cleaned, just parsed.

Reply:
@adisbladis Updated! Thanks for the feedback.